kernel/cClosure.ml

```diff
@@ -95,6 +94,7 @@ module type RedFlagsSig = sig
   val red_add_transparent : reds -> TransparentState.t -> reds
   val red_transparent : reds -> TransparentState.t
   val mkflags : red_kind list -> reds
+  val mkfullflags : red_kind list -> reds
```

Comment:
Probably should be exposed in the mli too.

Reply:
I am planning to rework this API anyways, splitting the flags for query from the one from creation. So I'd rather postpone.

Reply:
ok